quickstart/azure/tutorial-container-webserver.md

```diff
@@ -11,7 +11,8 @@ In this tutorial, we'll use JavaScript to deploy a simple webserver Container In
 
 ## Create a Virtual Machine with SSH access {#webserver}
 
-1.  In a new folder `webserver`, create an empty project with `pulumi new`. Make sure you have run `az login` or configured credentials for Azure.
+1.  Make sure you have run `az login` or configured credentials for Azure.
```

Comment:
This is covered as part of the `Prerequisites` above under the [Configure Azure credentials] link.  (and technically, it is not required depending on which approach you take for configuring your Azure credentials).

I'm not sure it makes sense to re-include everything there into this (and all other) tutorial texts.  We could consider make the "Prerequisites" more of proper steps here so that it's more clear they have to be completed.

I'd also generally love to maintain some horizontal consistency across these tutorials - so any change like this we would ideally make consistently across all tutorials.

Reply:
I agree re having horizontal consistency. Would you like me to create an include file for the Azure prerequisites and another for AWS and each other cloud used in a tutorial?

How about I change the text in the prerequisites section to say "Configure Azure credentials or Login to Azure"?